src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +51,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
```

Comment:
if all that grantLocationAccess_() does is call loadMap_() why not just remove grantLocationAccess and just only have the latter instead of chaining function calls?

Reply:
In future PRs there will be more functions called in grantLocationAccess_, I've held off on adding more here momentarily while I get the location permissions working